src/unitTest/PageParserTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+include_once '../parserPages/PageParser.php';
+include_once '../dataBase/XmlFeedModel.php';
+
+use PHPUnit\Framework\TestCase;
+
+class PageParserTest extends TestCase
+{
+    public function testPageParser()
+    {
+        $objectPage = new PageParser();
+        $sourceLink = 'https://usn.ubuntu.com/usn/rss.xml';
+        $haystack = $objectPage->getValuesInItem($sourceLink);
+        $this->assertIsArray($haystack);
+        $expected = $haystack[0];
```

Comment:
This might be wrong when the number of items on remote resource will be changed. You should not test the number of items, but you can use mocks instead.

Reply:
I expect there is a test that checks what happen if url is not accessible or did not return items.